packages/remix-fastify/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "@remix-run/fastify",
+  "version": "1.0.4",
+  "description": "Fastify server request handler for Remix",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/remix-run/remix",
+    "directory": "packages/remix-fastify"
+  },
+  "bugs": {
+    "url": "https://github.com/remix-run/remix/issues"
+  },
+  "dependencies": {
+    "@remix-run/node": "1.0.4"
```

Comment:
Shouldn't this package also have `@remix-run/server-runtime` as dependency? 🤔

Reply:
`@remix-run/node` has `@remix-run/server-runtime` as a dependency :)

Reply:
I think all of them should have `@remix-run/server-runtime` as a dependency then or at least a `peerDependency`